pkg/yurtadm/cmd/join/join.go

```diff
@@ -37,13 +37,29 @@ import (
 	"github.com/openyurtio/openyurt/pkg/util/kubernetes/kubeadm/app/constants"
 	"github.com/openyurtio/openyurt/pkg/util/kubernetes/kubeadm/app/discovery/token"
 	kubeconfigutil "github.com/openyurtio/openyurt/pkg/util/kubernetes/kubeadm/app/util/kubeconfig"
+	"github.com/openyurtio/openyurt/pkg/util/kubernetes/kubectl/pkg/util/i18n"
+	"github.com/openyurtio/openyurt/pkg/util/kubernetes/kubectl/pkg/util/templates"
 	"github.com/openyurtio/openyurt/pkg/yurtadm/cmd/join/joindata"
 	yurtphase "github.com/openyurtio/openyurt/pkg/yurtadm/cmd/join/phases"
 	yurtconstants "github.com/openyurtio/openyurt/pkg/yurtadm/constants"
 	yurtadmutil "github.com/openyurtio/openyurt/pkg/yurtadm/util/kubernetes"
 )
 
 var (
+	joinExample = templates.Examples(i18n.T(`
+		# Join the edge node to cluster.
+		yurtadm join 192.168.152.131:6443 --token=zffaj3.a5vjzf09qn9ft3gt --node-type=edge --discovery-token-unsafe-skip-ca-verification --v=5
```

Comment:
we should not use concrete ip address and token in the example

Reply:
OK